docs/download.md

```diff
@@ -6,6 +6,14 @@ sitemap:
   changefreq: "daily"
 ---
 
+::: info "Rolling" and "Regular" Releases
+
+We have "Rolling" and "Regular" releases to choose from. These differ only in the frequency of updates they receive. (See details below.) Please use whichever suits your preference.
```

Comment:
Perhaps link the Rolling and Regular to the appropriate sections?

```md
We have "[Rolling](#rolling-release)" and "[Regular](#regular-releases)" releases ...
```

Reply:
Good idea! I tested it, and it draws the eye slightly better (nice hyperlink coloring in the theme!), and funnels people directly to the release type they are most interested (by way of what they click on). It zooms down there (nice smooth scroll animation... I think I see why these fancy web frameworks are so well liked... !).

Auto-reloading dev environment is nice, too!

What a lovely repo this is to work in, if I dare say so myself!

Anyway, committed and pushed, thank you for the suggestion!